MixpanelDemo/MixpanelDemoTests/MixpanelNotificationTests.swift

```diff
@@ -14,93 +14,90 @@ import Nocilla
 
 class MixpanelNotificationTests: MixpanelBaseTests {
```

Comment:
I think we should test for all the possible fields in an in-app (including button's field and whether `buttons` is an array, for example).
We should also include tests for mini in-apps (probably a separate function)